### PR TITLE
Fixed #32964 -- Corrected 'setup'/'set up' usage in docs.

### PR DESCRIPTION
**Pull request by anorthall**
[Ticket #32964](https://code.djangoproject.com/ticket/32964) states that some instances of 'setup' (noun) and 'set up' (verb) are incorrectly used in the docs. I have been through every instance of 'setup' and 'set up' in the docs and changed them where required.

### CODE REVIEW

File: docs/intro/tutorial01.txt [hunk 1]
```diff
@@ -42,7 +42,7 @@ Creating a project
 ==================
 
 If this is your first time using Django, you'll have to take care of some
-initial setup. Namely, you'll need to auto-generate some code that establishes a
+initial set up. Namely, you'll need to auto-generate some code that establishes a
```
Comment (jacobtylerwalls) [inline docs/intro/tutorial01.txt:45]: Noun

File: docs/ref/contrib/auth.txt [hunk 2]
```diff
@@ -652,7 +652,7 @@ The following backends are available in :mod:`django.contrib.auth.backends`:
     .. method:: configure_user(request, user)
 
         Configures a newly created user.  This method is called immediately
-        after a new user is created, and can be used to perform custom setup
+        after a new user is created, and can be used to perform custom set up
```
Comment (jacobtylerwalls) [inline docs/ref/contrib/auth.txt:655]: noun

File: docs/ref/contrib/gis/install/index.txt [hunk 3]
```diff
@@ -243,7 +243,7 @@ the version of PostgreSQL in the Postgres.app you installed:
 
     export PATH=$PATH:/Applications/Postgres.app/Contents/Versions/X.Y/bin
 
-You can check if the path is set up correctly by typing ``which psql`` at a
+You can check if the path is setup correctly by typing ``which psql`` at a
```
Comment (jacobtylerwalls) [inline docs/ref/contrib/gis/install/index.txt:246]: verb

File: docs/releases/1.4.txt [hunk 4]
```diff
@@ -240,7 +240,7 @@ This method lets you create multiple objects more efficiently. It can result in
 significant performance increases if you have many objects.
 
 Django makes use of this internally, meaning some operations (such as database
-setup for test suites) have seen a performance benefit as a result.
+set up for test suites) have seen a performance benefit as a result.
```
Comment (jacobtylerwalls) [inline docs/releases/1.4.txt:243]: noun

File: docs/topics/cache.txt [hunk 5]
```diff
@@ -48,10 +48,10 @@ of caches that you don't directly control but to which you can provide hints
 Setting up the cache
 ====================
 
-The cache system requires a small amount of setup. Namely, you have to tell it
-where your cached data should live -- whether in a database, on the filesystem
-or directly in memory. This is an important decision that affects your cache's
-performance; yes, some cache types are faster than others.
+The cache system requires a small amount of set up. Namely, you have to tell
```
Comment (jacobtylerwalls) [inline docs/topics/cache.txt:51]: noun

File: docs/topics/testing/advanced.txt [hunk 6]
```diff
@@ -748,7 +748,7 @@ utility methods in the ``django.test.utils`` module.
 
 .. function:: setup_test_environment(debug=None)
 
-    Performs global pre-test setup, such as installing instrumentation for the
+    Performs global pre-test set up, such as installing instrumentation for the
```
Comment (jacobtylerwalls) [inline docs/topics/testing/advanced.txt:751]: noun

**Review by jacobtylerwalls — COMMENTED** [review]
Thanks for identifying these. I highlighted a few that I think should be reverted based on whether the use is as a noun or a verb.

**Comment by anorthall** [comment]
> Thanks for identifying these. I highlighted a few that I think should be reverted based on whether the use is as a noun or a verb.

@jacobtylerwalls thank you for having a look at this - I did struggle with a few of them! I have changed back the six that you specified and amended the commit.

File: docs/topics/cache.txt [hunk 7]
```diff
@@ -48,10 +48,10 @@ of caches that you don't directly control but to which you can provide hints
...
 ====================
 
-The cache system requires a small amount of setup. Namely, you have to tell it
-where your cached data should live -- whether in a database, on the filesystem
-or directly in memory. This is an important decision that affects your cache's
-performance; yes, some cache types are faster than others.
+The cache system requires a small amount of setup. Namely, you have to tell
+it where your cached data should live -- whether in a database, on the 
+filesystem or directly in memory. This is an important decision that affects
+your cache's performance; yes, some cache types are faster than others.
```
Comment (jacobtylerwalls) [inline docs/topics/cache.txt:54]: These lines no longer run over 79 chars, so these changes can be reverted.

File: docs/topics/db/models.txt [hunk 8]
```diff
@@ -424,8 +424,8 @@ Generally, :class:`~django.db.models.ManyToManyField` instances should go in
 the object that's going to be edited on a form. In the above example,
 ``toppings`` is in ``Pizza`` (rather than ``Topping`` having a ``pizzas``
 :class:`~django.db.models.ManyToManyField` ) because it's more natural to think
-about a pizza having toppings than a topping being on multiple pizzas. The way
-it's set up above, the ``Pizza`` form would let users select the toppings.
+about a pizza having toppings than a topping being on multiple pizzas. With the
```
Comment (jacobtylerwalls) [inline docs/topics/db/models.txt:427]:
```suggestion
about a pizza having toppings than a topping being on multiple pizzas. With the
```

**Review by jacobtylerwalls — COMMENTED** [review]
Noticed two small things on a second look. I'll mark this ready for checkin when addressed. 

(To mark it RFC I'm also supposed to ask for the commit message to take the same form as your PR title, e.g. ```Fixed #32964 -- Wrote a past tense fragment ending with a period.```) The ticket number part has to be up front to help the ticket tracker bot.

Thanks again for doing the audit.

**Comment by anorthall** [comment]
@jacobtylerwalls Done! :)

**Review by carltongibson — APPROVED** [review]
Hi @anorthall — Thanks for for this. 

I just moved the commit message to the _past tense_ and dropped the changes from `models.txt` — _The way it's set up…_ was fine as was. 🤔

Thanks, and welcome aboard! ⛵️

**Comment by carltongibson** [comment]
C.I. failures are unrelated.

**Pull request merged**